x-pack/plugins/maps/public/shared/layers/sources/kibana_tilemap_source/create_source_editor.js

```diff
@@ -46,19 +50,21 @@ export class CreateSourceEditor extends Component {
 
   render() {
 
-    if (this.state.url === null) {
+    if (this.state.url === null) {//still loading
       return null;
     }
 
+    const previewer = this.state.url ? (<EuiFieldText
```

Comment:
Putting the EuiFieldText into previewer  can be reverted because they are no longer needed

Reply:
what about empty string/undefined config?

Reply:
Showing the empty text input is more consistent with the other empty states. Otherwise, what is the the form help text describing?